src/modules/esl-media-query/core/conditions/media-query-condition.ts

```diff
@@ -0,0 +1,51 @@
+import {ALL, NOT_ALL} from './media-query-base';
+import type {IMediaQueryCondition} from './media-query-base';
```

Comment:
I suppose you wanted import interface IMediaQueryCondition

Reply:
Did you mean the `import` instead of `type import`?
If yes then there is an expected result here:
the interface is TS's entity to describe the type, so taking into account eslint rules on the project I should separate type imports for TS and real ES imports (which will be visible in the resulting bundle).